draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -5251,11 +5251,11 @@ Expect: 100-continue
    The example
 </t>
 <figure><artwork type="example">
-  Accept: audio/*; q=0.2, audio/basic
+  Accept: audio/mp3; q=0.2, audio/basic
```

Comment:
I don't understand why this is being changed.

Reply:
What's the use case for asking for `audio/*` vs. omitting the header field?

Reply:
The pattern I've seen is: `Accept: image/webp, */*; q=0.5`.  Though why `*/*` rather than `image/*` is not clear.  The point being that this is going into something that can handle a range of image formats - presumably too many to list in the request - but there is still a desire to express a preference for one or two particular formats.

s/image/audio/g as needed.

Reply:
What information does that header contain that isn't conveyed by `Accept: image/webp`? The server is already explicitly allowed to ignore the header as it sees fit.

Reply:
The original motivator was external viewers like xv that were configured in the browser to handle image/* as an extension (the browser itself had no idea what image formats were actually acceptable -- it was just instructed to accept all of them). This can also show up in audio-only environments (screen readers) where an audio alternative is preferred if available.

Also, originally, the server was not allowed to ignore; it was just that only a few implemented negotiation, and even then only for specific resources. That spec text is just trying to capture reality that the user agent has no control over which resources are actually negotiable, and when that negotiation happens, which makes it appear like the server can ignore. We tried (and failed) to address that with 406.

Reply:
Did I mention that I hate content negotiation?